docs/source/notebooks/wps_generate_climos.ipynb

```diff
@@ -0,0 +1,388 @@
+       "\u001b[0;34m\u001b[0m    \u001b[0msplit_vars\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\n",
+       "\u001b[0;34m\u001b[0m    \u001b[0msplit_intervals\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\n",
+       "\u001b[0;34m\u001b[0m    \u001b[0mloglevel\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'INFO'\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\n",
+       "\u001b[0;34m\u001b[0m    \u001b[0mclimo\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\n",
+       "\u001b[0;34m\u001b[0m    \u001b[0mresolutions\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\n",
+       "\u001b[0;34m\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+       "\u001b[0;31mDocstring:\u001b[0m\n",
+       "Generate files containing climatological means from input files of daily, monthly, or yearly data that adhere to the PCIC metadata standard (and consequently to CMIP5 and CF standards).\n",
+       "\n",
+       "Parameters\n",
+       "----------\n",
+       "netcdf : ComplexData:mimetype:`application/x-netcdf`, :mimetype:`application/x-ogc-dods`\n",
+       "    NetCDF file\n",
+       "operation : {'mean', 'std'}string\n",
+       "    Operation to perform on the datasets\n",
+       "climo : {'6190', '7100', '8110', '2020', '2050', '2080'}string\n",
+       "    Year ranges\n",
+       "resolutions : {'all', 'yearly', 'seasonal', 'monthly'}string\n",
+       "    Temporal Resolutions\n",
+       "convert_longitudes : boolean\n",
+       "    Transform longitude range from [0, 360) to [-180, 180)\n",
+       "split_vars : boolean\n",
+       "    Generate a separate file for each dependent variable in the file\n",
+       "split_intervals : boolean\n",
+       "    Generate a separate file for each climatological period\n",
+       "dry_run : boolean\n",
+       "    Checks file to ensure compatible with process\n",
+       "loglevel : {'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'}string\n",
+       "    Logging level\n",
+       "\n",
+       "Returns\n",
+       "-------\n",
+       "output : ComplexData:mimetype:`application/metalink+xml; version=4.0`\n",
+       "    Metalink object between output files\n",
+       "dry_output : ComplexData:mimetype:`application/metalink+xml; version=4.0`\n",
+       "    Metalink object between dry output files\n",
+       "\u001b[0;31mFile:\u001b[0m      ~/code/birds/thunderbird/</tmp/thunderbird-venv/lib/python3.8/site-packages/birdy/client/base.py-0>\n",
+       "\u001b[0;31mType:\u001b[0m      method\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Check info on `generate_climos` process\n",
+    "thunderbird.generate_climos?"
```

Comment:
Add `# NBVAL_IGNORE_OUTPUT` to this cell to prevent `nbval` from incorrectly reading it.